package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "precommit": "lint-staged",
     "tsc": "tsc",
+    "postinstall": "tsc --outDir ./lib",
```

Comment:
Could you explain, why it's a good idea, to have this `postinstall` script here? I thought it's run after `npm install` which doesn't seem to make too much sense or am I missing something?

Reply:
Sorry about that, I was using that Github fork for my own project hence why I have a postInstall script there. I have since removed it.